Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: all
+all: .installed.cfg
+
+py3/bin/buildout: py3/bin/pip3 requirements.txt
+	./py3/bin/pip3 uninstall -y setuptools
+	./py3/bin/pip3 install -IUr requirements.txt
+	./py3/bin/pre-commit install
+
+py3/bin/pip3:
+	python3 -m venv py3
+
+.installed.cfg: py3/bin/buildout
+	./py3/bin/buildout -c dev.cfg
```

Comment:
```suggestion
	./py3/bin/buildout
```

Reply:
You mean getting rid of dev.cfg and only using buildout.cfg? fine with me.